app/vmalert/rule/recording.go

```diff
@@ -202,6 +209,17 @@ func (rr *RecordingRule) exec(ctx context.Context, ts time.Time, limit int) ([]p
 	return tss, nil
 }
 
+func (rr *RecordingRule) logDebugf(at time.Time, format string, args ...any) {
+	if !rr.Debug {
+		return
+	}
+	prefix := fmt.Sprintf("DEBUG recording rule %q:%q (%d) at %v: ",
+		rr.GroupName, rr.Name, rr.RuleID, at.Format(time.RFC3339))
```

Comment:
```suggestion
	prefix := fmt.Sprintf("DEBUG recording rule %q/%q:%q (%d) at %v: ",
		rr.File, rr.GroupName, rr.Name, rr.RuleID, at.Format(time.RFC3339))
```
We were missing rule file name.

Please also fix this for alerting rule
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/0e142e4e11daa7d8ae5f246b90ebc0f99bac3178/app/vmalert/rule/alerting.go#L210-L212

```
	prefix := fmt.Sprintf("DEBUG alerting rule %q/%q:%q (%d) at %v: ",
		ar.File, ar.GroupName, ar.Name, ar.RuleID, at.Format(time.RFC3339))
```